Add post introducing Dragonwell (#593)
AdoptOpenJDK/blog: content/blog/adoptopenjdk-welcomes-dragonwell/index.md

---
title: AdoptOpenJDK Welcomes Dragonwell
date: "2020-11-10T12:00:00+00:00"

> author: andreasahlenstorf

Reviewer: I'm wondering whether this particular update should come from the tsc user?

description: With Alibaba Dragonwell, another flavour of OpenJDK is being built at and distributed by AdoptOpenJDK. Dragonwell is a TCK-certified implementation of Java for large-scale application deployments on Alibaba Cloud and elsewhere.
tags:
  - announcement
---

> AdoptOpenJDK is happy to announce that Alibaba decided to join forces with us by building and distributing their [Dragonwell JDK](http://dragonwell-jdk.io/) on our infrastructure. For our users, this means more choice, mainly because Dragonwell offers some exciting features that are unique to Dragonwell. One example is [Wisp](https://github.com/alibaba/dragonwell8/wiki/Wisp-Documentation) that implements coroutines on the JVM.

Reviewer 2: Would Alibaba agree to this being a joint announcement?  Consider:

```suggestion
AdoptOpenJDK and Alibaba are happy to announce we have joined forces to build, test, and distribue [Dragonwell JDK](http://dragonwell-jdk.io/) on the AdoptOpenJDK infrastructure. For our users, this means more choice, including some exciting new features that are unique to Dragonwell. One example is [Wisp](https://github.com/alibaba/dragonwell8/wiki/Wisp-Documentation) that implements coroutines on the JVM.
```

> 

Author: Something for the post about Dragonwell.

> Over the next weeks, the first releases of Dragonwell 8 and 11 are going to appear on AdoptOpenJDK's website alongside our existing binary offerings. Dragonwell can be consumed like any other binary release by AdoptOpenJDK. The Dragonwell team will also introduce Dragonwell JDK and its unique features in a series of blog posts. Stay tuned!

Reviewer 2: ```suggestion
Over the next weeks, the first releases of Dragonwell 8 and 11 are going to appear on AdoptOpenJDK's website alongside existing binary offerings. Dragonwell can also be consumed like any other binary release built at AdoptOpenJDK through the [AdoptOpenJDK API](https://api.adoptopenjdk.net/). The Dragonwell team will also introduce Dragonwell JDK and its unique features in a series of blog posts. Stay tuned!
```

> Over the next weeks, the first releases of Dragonwell 8 and 11 are going to appear on AdoptOpenJDK's website alongside our existing binary offerings. Dragonwell can be consumed like any other binary release by AdoptOpenJDK. The Dragonwell team will also introduce Dragonwell JDK and its unique features in a series of blog posts. Stay tuned!

Reviewer 3: Here is a brief introduction for the available customized features in Dragonwell: 
• Multi-tenant: running multiple apps in the same instance safely
• Wisp: building the user-mode thread in Java, allowing Java developers to write code in synchronous, but gain the performance of asynchronous.
• JWarmup: priming online application for speed
• ElasticHeap:  scaling JVM's memory utilization dynamically according to the application's needs.
• JFR extension: extending more JFR events inside JVM for efficient problem diagnostics.

Author: If left those out. That would warrant further explanations.

> Dragonwell is a flavour of OpenJDK (like AdoptOpenJDK!). It is a TCK-certified [[1](#fn-tck)<a name="fn-tck-ret"></a>] implementation of Java for large-scale application deployments on Alibaba Cloud and elsewhere. As such, it is a drop-in replacement for any other OpenJDK variant and compatible with all the tools you know, like JDK Flight Recorder and JDK Mission Control.

Reviewer 3: Dragonwell's relationship with openjdk:

Alibaba Dragonwell is certified as compatible with the Java SE standard. 
Alibaba Dragonwell is a "friendly fork" under the same licensing terms as the upstream OpenJDK project. Alibaba is committed to collaborating closely with the OpenJDK community and intends to bring as many customized features as possible from Alibaba Dragonwell to the upstream.

Author: Included with slight changes.

---

> 1. Alibaba<a name="fn-tck"></a> performs the TCK certification, not AdoptOpenJDK. [↩︎](#fn-tck-ret "Go back to footnote 1 in the text")

Author: Dragonwell is linked above. The footnote is there to avoid any confusion regarding TCK usage.
